bonjour.js

```diff
@@ -15,10 +15,33 @@
  * limitations under the License.
  */
 
-var express = require('express');
+// zipkin-related
+const express = require('express');
+const {Tracer, ExplicitContext, BatchRecorder} = require('zipkin');
+const zipkinMiddleware = require('zipkin-instrumentation-express').expressMiddleware;
+
+const ctxImpl = new ExplicitContext();
+const {HttpLogger} = require('zipkin-transport-http');
+
+const recorder = new BatchRecorder({
+	logger: new HttpLogger({
+		endpoint: 'http://zipkin-query:9411/api/v1/spans'
```

Comment:
We should use ZIPKIN_URL as the other services. This env var should be declared inside Dockerfile as I commented here: https://github.com/redhat-helloworld-msa/aloha/pull/7#discussion_r75034888